pom.xml

```diff
@@ -223,6 +223,15 @@
             </lifecycleMappingMetadata>
           </configuration>
         </plugin>
+
+        <!-- Inject the "incubating" into the output filename -->
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-assembly-plugin</artifactId>
+          <configuration>
+            <finalName>edgent-incubating-${project.version}</finalName>
```

Comment:
Makes sense, thanks.  I also noticed that the current bundles have an apache prefix ("apache-edgent-<ver>....tgz").  Don't recall if that's a best practice, I seem to have a recollection of Justin noting something about it :-)  Your thoughts?   

We also currently create .md5 and .sha files for distribution.  Are they missing or do they only get created as part of a different task (I used mvn install -Papache-release)?  

Previously we only created a tgz bundle.  Now it looks like only a zip?  Seems OK but wanted to double check.